Orange/data/variable.py

```diff
@@ -35,7 +35,7 @@ def make_variable(cls, compute_value, *args):
         return cls(*args, compute_value=compute_value)
     else:
         # For compatibility with old pickles
-        return cls(*args)
+        return cls(*tuple(x for i, x in enumerate(args) if i != 2))
```

Comment:
What about this? @lanzagar, I suppose this should be OK?

```python
# For compatibility with old pickles: remove the second arg if it's bool
# `compute_value` (args[3]) can't be bool, so this should be safe
if len(args) >= 2 and isinstance(args[2], bool):
    args = args[:2] + args[3:]
return cls(*args)
```

Reply:
if `len(args) == 2` this raises an IndexError.